src/rdb/tests/README

```diff
@@ -0,0 +1,36 @@
+The rdb_test_runner.py script is now out of date and will be
+removed at a future time when this rdbt test will be converted
+to an automated unit test.
+
+Manual execution instructions:
+
+From the command line the tests are run with:
+
+==== server: start on S server ranks:
+orterun -np <S> --hostfile <file_containing_server_hostnames> --map-by-node
+ --enable-recovery -x LD_LIBRARY_PATH daos_server
+ -o <builddir>/utils/config/examples/daos_server_unittests.yml
+ start -d ./ -t 1 -m vos,rdb,rsvc,mgmt,rdbt
+
```

Comment:
Sorry, forgot to ask if daos_agent is required. We do need it for attaching to work, right?

Reply:
Yes daos_agent is needed. Perhaps I could land this PR first, and include the README update to include the daos_agent item in the next DAOS PR. Another DAOS PR is needed  to update the raft submodule after Raft PR https://github.com/daos-stack/raft/pull/36 lands that will expose the raft_electon_start() function. The DAOS PR will need to remove the extern declaration of raft_election_start() from src/rdb/rdb_raft.c

Or, I could just wait for raft PR 36 to land, and update this patch with the necessary DAOS changes including the README. What do you think?

Reply:
Either way is fine with me.

Reply:
Raft PR 36 is ready for landing, so I'll let that happen first and do the remaining steps in this PR for a single landing.